test/integration/ui-extension-integration.js

```diff
@@ -70,4 +70,35 @@ export default function uiExtensionTests(t, space) {
         t.equals(uiExtension.extension.src, 'https://awesome.extension', 'src')
       })
   })
+
+  t.test('Filter UI extensions by ID', async (t) => {
+    const idOne = 'idOne'
+    const idTwo = 'idTwo'
+
+    const extensionOne = await space.createUiExtensionWithId(idOne, {
+      extension: {
+        name: 'Awesome extension!',
+        src: 'https://awesome.extension',
+        fieldTypes: [{ type: 'Symbol' }],
+      },
+    })
+
+    const extensionTwo = await space.createUiExtensionWithId(idTwo, {
+      extension: {
+        name: 'Another awesome extension!',
+        src: 'https://anotherawesome.extension',
+        fieldTypes: [{ type: 'Text' }],
+      },
+    })
+
+    const environment = await space.getEnvironment('master')
+
+    const extensions = await environment.getUiExtensions({ 'sys.id[in]': idTwo })
```

Comment:
What do we run it against? Don't we have to mock the API behavior somewhere?

Reply:
So far as I can tell these are actually end-to-end tests. It doesn't appear that we do any mocking.